packages/cli/src/commands/env/pull.ts

```diff
@@ -48,7 +48,8 @@ export default async function pull(
   environment: ProjectEnvTarget,
   opts: Partial<Options>,
   args: string[],
-  output: Output
+  output: Output,
+  cwd: string = process.cwd()
```

Comment:
```suggestion
  cwd: string,
```

This can be a required param now, right?

Reply:
Good call, yeah